### DISCUSSION TITLE
Couldn't read out-dir-path/bindings.rs

### DISCUSSION THREAD

**Discussion by jcornaz** [boozook/playdate#182] [Q&A] [answered]
Hi, 

`cargo playdate package` works like a charm from Linux.

But `cargo playdate package --device` from linux and `cargo playdate package` from windows fail with a similar error message:

```
error: couldn't read ~/my-project/target/thumbv7em-none-eabihf/debug/build/playdate-sys-211309a35f5c8688/out/pd2.1.1-pd-000000000.rs: No such file or directory (os error 2)
  --> ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/playdate-sys-0.2.10/src/lib.rs:50:2
   |
50 |     include!(env!("PD_BINDINGS_PATH"));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include` (in Nightly builds, run with -Z macro-backtrace for more info)
```

I don't get it. Do you have any idea?

**Comment by boozook** [comment, marked as answer]
Just update needed to rebuild pre-built bindings.
Look, your configuration seems to "don't generate bindings for me, just use what included in the package" and there is no 2.1.1 in the package yet.

First solution - build it for yourself, second is to update it here. For second I [run the action](https://github.com/boozook/playdate/actions/runs/6887331937).


- - -

Hm, [there's no update](https://github.com/boozook/playdate/actions/runs/6887331937/job/18734425137#step:4:26) 🤔

Ok, it's already updated (#181), but [had not](https://github.com/boozook/playdate/actions/runs/6844188079) published.
[Done](https://github.com/boozook/playdate/actions/runs/6887393558/job/18734524583#step:5:184).

**Reply by boozook** [reply]
@jcornaz jut ping me if something wrong or doesn't work.

**Reply by boozook** [reply]
Well, I've released crate to crate.io, so you could do cargo update and it should work, I suppose.

**Reply by jcornaz** [reply]
Seems to work so far. Thank you very much for the help.
